Tests/AppTests/SearchShowModelTests.swift

```diff
@@ -14,9 +14,10 @@
 
 @testable import App
 
+import SnapshotTesting
 import XCTVapor
 
-class SearchShowModelTests: XCTestCase {
+class SearchShowModelTests: AppTestCase {
```

Comment:
Better to use `SnapshotTestCase` here instead, it pins dates and makes it easy to switch on recorded if needed.

Note though that changing from `XCTestCase` to `AppTestCase` (or `SnapshotTestCase`, which inherits from `AppTestCase`) means all tests now run through a db tearDown and setUp phase, making them much slower (0.90s vs 0.01s).

It's not a huge deal with only six other tests in this file but it would be better to leave this as `XCTestCase` and pull out the test that needs the snapshot into its own class (even if it's in the same file). E.g.

```swift
class SearchShowModelSnapshotTests: SnapshotTestCase {
    func test_SearchShow_Model_canonicalURLAllowList() throws {
        try app.test(.GET, "search?query=alamo&page=2&utm_campaign=test&utm_source=email", afterResponse: { res in
            XCTAssertEqual(res.status, .ok)

            assertSnapshot(matching: res.body.string, as: .html)
        })
    }
}
```

That way if more tests are added that don't need the db they don't automatically pay the setup cost.